classes/SpotifyFeed.php

```diff
@@ -41,6 +41,23 @@ private function verifySong(string $song_id, string $user){
     }
     
     /**
+     * Accept a valid username and a specific song's id
+     * and save it into the user's liked songs list
+     *
+     * @param string $song_id The Spotify song id code
+     * @param string $user The current user's username
+     */
+    public function saveSong(string $song_id, string $user){
+        $sql = "INSERT INTO likes(user, songId) VALUES
+            (:user, :songId)";
+        $stmt = $this->db->prepare($sql);
+        $result = $stmt->execute(["user" => $user,  "songId" => $song_id,]);
+        if(!$result) {
```

Comment:
same PSR-2... I won't leave any more PSR-2 comments, so not to pollute this, but there should be a space between `if` and `(` like: `if (`.